src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -119,54 +154,28 @@ Allow: /
 Sitemap: https://<YOUR SITE>/sitemap-index.xml
 ```
 
-To generate `robots.txt` dynamically, add a file named `robots.txt.ts` with the following code:
+You can also generate `robots.txt` dynamically if you want to reuse the `site` value from `astro.config.mjs`.
+To do so, create a `src/pages/robots.txt.ts` file with the following code:
```

Comment:
```suggestion
To generate a `robots.txt` file dynamically using the `site` value from `astro.config.mjs`, create the file `src/pages/robots.txt.ts` and add the following code:
```

Reply:
This removes the explanation for _why_ you might do this and almost makes it sound like you should use this dynamic approach as well as the static approach — which you shouldn’t. Maybe that should be clearer?

```suggestion
If you want to reuse the `site` value from `astro.config.mjs`, you can also generate `robots.txt` dynamically.
Instead of using a static file in the `public/` directory, create a `src/pages/robots.txt.ts` file and add the following code:
```

Reply:
Yeah, I was worried about whether the why was clear enough in my suggestion. I didn't think it made it sound like it would do both, but if my suggestion also introduces that, then I am happy to withdraw my suggestion.

Mostly, I was trying to avoid "to do so" because that feels like we really should be able to write around that. :smile: